src/ages/distributions/ageFourDistribution/ageFourDistribution.ts

```diff
@@ -0,0 +1,33 @@
+import { EpochConfig } from "../../ages.types";
+import { AgeDistribution } from "../distributions.types";
+import { providers } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+import marketsRepartition from "./marketsRepartition";
+import { weightedDistribution } from "../weightedDistribution";
+import { PercentMath } from "@morpho-labs/ethers-utils/lib/maths";
+
+export const ageFourDistribution = async (
+  ageConfig: AgeDistribution,
+  { finalTimestamp, initialTimestamp, number, snapshotBlock, totalEmission }: EpochConfig,
+  provider?: providers.Provider
+) => {
+  if (!snapshotBlock) throw Error(`Cannot distribute tokens for epoch ${number}: no snapshotBlock`);
```

Comment:
Do we name age4 and epoch nb 10 the same thing?

Reply:
Yes, the refactor to remove the idea of epochs includes a lot of changes. So the repo is going to keep 1 epoch per age, named `ageX`